porcupine/plugins/highlight/tree-sitter-token-mappings/python.yml

```diff
@@ -202,3 +255,132 @@ token_mapping:
     type: Token.Name.Builtin
     vars: Token.Name.Builtin
     zip: Token.Name.Builtin
+
+    # builtin.Pseudo
```

Comment:
Would this be a good category for `reveal_type()`?

Reply:
What do you mean?

Reply:
mypy has a "fake" built-in `reveal_type()`. It would be good to have it highlighted, even though it's technically not special to Python in any way. Currently there's no indication that you're doing anything right when trying to add `reveal_type()`, as you get linter errors and no sign that it's special.